ansible_collections/arista/avd/roles/eos_designs/templates/underlay/interfaces/ethernet-interfaces.j2

```diff
@@ -37,6 +37,9 @@ ethernet_interfaces:
     isis_metric: 50
     isis_network_point_to_point: true
 {%         endif %}
+{%         if link.link_tracking_groups is arista.avd.defined %}
```

Comment:
This should also be added for `underlay_l2` type below

Reply:
This is for complete MLAG system right?

But anyway, I will add there.